src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -56,7 +56,7 @@ public Iterable findAll() {
 	@GetMapping("/author/{author}")
 	public Book findByAuthor(@PathVariable String author) {
 		return bookRepository.findByAuthor(author)
-				.orElseThrow(() -> new BookNotFoundException(NotificationCode.DATA_NOT_FOUND));
+				.orElseThrow(() -> new DataNotFoundException(NotificationCode.DATA_NOT_FOUND));
```

Comment:
By changing this exception, we can not identify anymore if it was a Book or a User the model that was not found. Maybe you can use this NotificationCode class to specify which model was not found...